code/modules/projectiles/guns/shotguns.dm

```diff
@@ -641,9 +641,9 @@ can cause issues with ammo types getting mixed up during the burst.
 	aim_fire_delay = 1
 
 	fire_delay = 17.5
-	accuracy_mult = 1.4
+	accuracy_mult = 1.45
```

Comment:
had an irremoveable barrel with stats for some reason, nice one high, same for the scatter

Reply:
same happens to stock as well